<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyPressWidget</class>
 <widget class="QWidget" name="KeyPressWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>125</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	#this does break dark mode stupid me
	#background-color: white;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>X.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="modBox">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="modBoxButtons">
        <item>
         <widget class="QPushButton" name="ctrlMod">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton:selected {
		color: white;
}</string>
          </property>
          <property name="text">
           <string>Ctrl</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shiftMod">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Shift</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="altMod">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Alt</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="metaMod">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Meta</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="customMod">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Custom...</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="customModBox">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Custom Modifier: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="CKeySequenceEdit" name="customSequenceEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="clearButton2">
           <property name="icon">
            <iconset theme="edit-clear">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="keyBox">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="CKeySequenceEdit" name="keySequenceEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="icon">
         <iconset theme="edit-clear">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="moveUp">
     <property name="icon">
      <iconset theme="go-up">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="moveDown">
     <property name="icon">
      <iconset theme="go-down">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delButton">
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CKeySequenceEdit</class>
   <extends>QKeySequenceEdit</extends>
   <header location="global">gui.customwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>delButton</sender>
   <signal>clicked()</signal>
   <receiver>KeyPressWidget</receiver>
   <slot>_remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>807</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveDown</sender>
   <signal>clicked()</signal>
   <receiver>KeyPressWidget</receiver>
   <slot>_moveDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveUp</sender>
   <signal>clicked()</signal>
   <receiver>KeyPressWidget</receiver>
   <slot>_moveUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customMod</sender>
   <signal>clicked(bool)</signal>
   <receiver>customModBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton2</sender>
   <signal>clicked()</signal>
   <receiver>customSequenceEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>keySequenceEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>658</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>623</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_remove()</slot>
  <slot>_moveUp()</slot>
  <slot>_moveDown()</slot>
 </slots>
</ui>
